src/dotnet/Chat/ChatPermissionsExt.cs

```diff
@@ -35,10 +35,7 @@ public static void Require(this ChatPermissions available, ChatPermissions requi
     }
 
     public static Exception NotEnoughPermissions(ChatPermissions? required = null)
-    {
-        var message = "You can't perform this action: not enough permissions.";
-        return required.HasValue
-            ? new SecurityException($"{message} Requested permission: {required.Value}.")
-            : new SecurityException(message);
-    }
+        => required.HasValue
```

Comment:
what if we make StandardErrorEnoughPermission(string? message = null) instead? This will allow to get rid off ternary operator

Reply:
done